addons/bestja_frontend_ucw/templates/pages/about_us.xml

```diff
@@ -0,0 +1,187 @@
+<?xml version="1.0" encoding="utf-8"?>
```

Comment:
Since those are static page fixtures, intended for production use, that do not seem to contain any programming logic, there is probably no reason to put them into the same module as, for example, CSS rules. Those static pages should either be removed from the repository or put in a separate module.